test/zipkin-simple.test.js

```diff
@@ -234,26 +230,26 @@ describe("Zipkin client", function () {
 					done()
 				})
 
-				Client.clientRecv(traceData, {
+				Client.sendClientRecv(traceData, {
```

Comment:
don't mind me much I don't usually write javascript so I've no idea what idiomatic is. Just it seems odd to carry around the span name multiple times.